source/_posts/2018-10-23-release-v11.0.0.md

```diff
@@ -0,0 +1,625 @@
+  * `console.time()` will no longer reset a timer if it already exists. [#20442](https://github.com/nodejs/node/pull/20442)
+* Dependencies
+  * V8 has been updated to 7.0. [#22754](https://github.com/nodejs/node/pull/22754)
+* `fs`
+  * The `fs.read()` method now requires a callback. [#22146](https://github.com/nodejs/node/pull/22146)
+  * The previously deprecated `fs.SyncWriteStream` utility has been removed.[#20735](https://github.com/nodejs/node/pull/20735)
+* `http`
+  * The `http`, `https`, and `tls` modules now use the WHATWG URL parser by default. [#20270](https://github.com/nodejs/node/pull/20270)
+* General
+  * Use of `process.binding()` has been deprecated. Userland code using `process.binding()` should re-evaluate that use and begin migrating. If there are no supported API alternatives, please open an issue in the Node.js GitHub repository so that a suitable alternative may be discussed.
+  * An experimental implementation of `queueMicrotask()` has been added. [#22951](https://github.com/nodejs/node/pull/22951)
+* Internal
+  * Windows performance-counter support has been removed. [#22485](https://github.com/nodejs/node/pull/22485)
+  * The `--expose-http2` command-line option has been removed. [#20887](https://github.com/nodejs/node/pull/20887)
+* Timers
+  * Interval timers will be rescheduled even if previous interval threw an error. [#20002](https://github.com/nodejs/node/pull/20002)
+  * `nextTick` queue will be run after each immediate and timer. [#22842](https://github.com/nodejs/node/pull/22842)
+* `util`
+  * The WHATWG `TextEncoder` and `TextDecoder` are now globals. [#22281](https://github.com/nodejs/node/pull/22281)
+  * `util.inspect()` output size is limited to 128 MB by default. [#22756](https://github.com/nodejs/node/pull/22756)
+  * A runtime warning will be emitted when `NODE_DEBUG` is set for either `http` or `http2`. [#21914](https://github.com/nodejs/node/pull/21914)
+-->
+
+### 주요 변경사항
+
+* 빌드
+  * FreeBSD 10은 지원되지 않습니다. [#22617](https://github.com/nodejs/node/pull/22617)
+* `child_process`
+  * `windowsHide` 옵션의 기본값이 `true`로 변경되었습니다. [#21316](https://github.com/nodejs/node/pull/21316)
+* `console`
+  * 타이머가 리셋되지 않았다면 `console.countReset()`은 경고를 출력합니다. [#21649](https://github.com/nodejs/node/pull/21649)
+  * 만약 타이머가 존재하면 `console.time()`은 타이머를 리셋하지 않습니다. [#20442](https://github.com/nodejs/node/pull/20442)
+* 의존
+  * V8이 7.0으로 업데이트되었습니다. [#22754](https://github.com/nodejs/node/pull/22754)
+* `fs`
+  * `fs.read()`메서드는 콜백을 필요로 합니다. [#22146](https://github.com/nodejs/node/pull/22146)
+  * 삭제 예정이었던 `fs.SyncWriteStream` 유틸리티는 삭제되었습니다. [#20735](https://github.com/nodejs/node/pull/20735)
+* `http`
+  * `http`, `https`, `tls` 모듈은 기본적으로 WHATWG URL 파서를 사용합니다. [#20270](https://github.com/nodejs/node/pull/20270)
+* 일반
+  * `process.binding()`은 삭제예정입니다. `process.binding()`을 사용하는 코드에서는 이식과 재계산을 할 예정입니다. 만약 지원되는 API의 대안이 없다면, Node.js 깃허브 저장소에 이슈를 생성해주세요. 그럼 적절한 대안에 대한 논의가 있을 것입니다.
```

Comment:
> Use of `process.binding()` has been deprecated. Userland code using `process.binding()` should re-evaluate that use and begin migrating. If there are no supported API alternatives, please open an issue in the Node.js GitHub repository so that a suitable alternative may be discussed.
```suggestion
  * `process.binding()`은 삭제예정입니다. 코드에 `process.binding()`을 사용하신다면 재확인하시고 이관을 시작하셔야 합니다. 만약 지원되는 API의 대안이 없다면, Node.js 깃허브 저장소에 이슈를 생성해주세요. 그럼 적절한 대안에 대한 논의가 있을 것입니다.
```